composer.json

```diff
@@ -93,27 +91,32 @@
         "prestashop/statssearch": "^2",
         "prestashop/statsstock": "^2",
         "prestashop/statsvisits": "^2",
-        "prestashop/translationtools-bundle": "^3",
+        "prestashop/translationtools-bundle": "dev-sf4-support",
         "prestashop/welcome": "^6",
-        "sensio/distribution-bundle": "^5.0.19",
-        "sensio/framework-extra-bundle": "^5.0.0",
+        "sensio/distribution-bundle": "^5.0",
+        "sensio/framework-extra-bundle": "^5.1",
         "shudrum/array-finder": "^1.1.0",
         "smarty/smarty": "^3.1.31",
         "soundasleep/html2text": "^0.5.0",
-        "swiftmailer/swiftmailer": "~5.4.3",
+        "swiftmailer/swiftmailer": "~6.2",
         "symfony/monolog-bundle": "^3.1.0",
         "symfony/polyfill-apcu": "^1.0",
         "symfony/polyfill-php73": "^1.10",
```

Comment:
Are we really using this component? (same for acpu polyfill?)

Reply:
We probably don't, but how can we be sure now it's here?

Reply:
There are 5 functions / Class this polyfill is taking in consideration, we can maybe try to find if it used 🤔

Reply:
We are, yes. Look at the PR which introduced it. :)

https://github.com/PrestaShop/PrestaShop/pull/11510